ova/generate_ova.sh

```diff
@@ -47,12 +47,20 @@ function build_ova() {
   local OVF_VM="wazuh${OVA_VERSION}.ovf"
   local OVA_FIXED="wazuh${OVA_VERSION}-fixed.ova"
   local OVA_VMDK="wazuh${OVA_VERSION}-disk001.vmdk"
+  local ELK_MAJOR=`echo ${ELK_VERSION}|cut -d"." -f1`
```

Comment:
`local` syntax is `bash` specific syntax. If this script is executed in a host where bash isn't installed, it won't work. The best solution is to set the _shebang_ to `#!/bin/sh` and use `sh` syntax which it is available in all Unix hosts.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/223/commits/63e8681803657ce52428b8a65324cae5a2cc696c